weblate/machinery/forms.py

```diff
@@ -217,3 +217,19 @@ class DeepLMachineryForm(KeyURLMachineryForm):
         label=pgettext_lazy("Automatic suggestion service configuration", "API URL"),
         initial="https://api.deepl.com/v2/",
     )
+    formality = forms.CharField(
+        label=pgettext_lazy(
+            "Automatic suggestion service configuration", "Default formality"
+        ),
+        help_text=gettext_lazy(
+            "Uses the specified formality if language is not specified as (in)formal"
+        ),
+        widget=forms.Select(
+            choices=(
+                ("default ", "Default"),
+                ("prefer_more", "Formal"),
+                ("prefer_less", "Informal"),
+            )
+        ),
+        initial="default",
```

Comment:
```suggestion
        initial="default",
        required=False,
```

Without this, existing configuration will be suddenly invalid (and this is why the tests fail).